docs/docs/overview/quickstart/get-started-with-dotnet.md

```diff
@@ -0,0 +1,359 @@
+- Set up a notification template.
+- Create subscribers and update subscriber information.
+- Send your first notification.
+- Send notifications via topics.
+
+## Requirements
+
+To follow the steps in this quickstart, you'll need:
+
+- A Novu account. [Sign up for free](http://web.novu.co) if you don’t have one yet.
+- A working .NET development environment with a .NET version of .NET Core 2.0+ and .NET Framework of 4.6.1+.
+
+You can also [view the completed code](https://github.com/novuhq/novu-dotnet-quickstart) of this quick start in a GitHub repo.
+
+
+## Install Novu .NET SDK
+
+The [.NET SDK](https://github.com/novuhq/novu-dotnet) provides a fluent and expressive interface for interacting with Novu's API and managing notifications.
+
+Now install the Novu .NET SDK by running the following command in your terminal:
+
+```bash
+dotnet add package Novu
+```
+
+Otherwise, modify your .csproj file add the following to it:
+
+```xml
+<ItemGroup>
+    <PackageReference Include="Novu" Version="*" />
+</ItemGroup>
+```
+
+Next, install the SDK by running the following `dotnet` command:
+
+```bash
+dotnet restore
+```
+
+## Initialize & Configure the Novu SDK
+
+Create a new file, `Program.cs` in your application and add the following code to it:
+
+Program.cs:
+```c#
+using Novu.DTO;
+using Novu.DTO.Topics;
+using Novu.Models;
+using Novu;
+
+
+ var novuConfiguration = new NovuClientConfiguration
+{
+    ApiKey = "<YOUR_NOVU_API_KEY>",
+};
+
+var novu = new NovuClient(novuConfiguration);
+
+var subscribers = await novu.Subscriber.GetSubscribers();
+```
+
+Replace the `ApiKey`’s value with the authentic key from the **API Key** section of your [Novu Dashboard](https://web.novu.co/settings) and be sure to use your specific Novu instance URL.
+
+:::info
+Please do not hardcode your credentials in a file in production. Use environment variables instead.
+:::
+
+## Set Up A Channel Provider
+
+A channel provider is a service that provides one or more notification functionality such as sending an email, SMS, push notification etc. Our [integration store](https://web.novu.co/integrations) includes four channels: Email, SMS, Chat, and Push. These channels have multiple providers associated with them.
+
+| Channel | Providers                                                           |
+| ------- | ------------------------------------------------------------------- |
+| Email   | MailGun, Mandrill, MailJet, Amazon SES, Sendgrid, Postmark, Netcore |
+| SMS     | Twilio, Amazon SNS, Plivo, SMS, SMSCentral, Kannel, Infobip, Termii |
+| Chat    | Mattermost, Slack, Microsoft Teams, Discord                         |
+| Push    | FCM, APNS, Expo                                                     |
+
+Only one provider can be **active** per **channel**. Connect any of your favorite providers to get started. The email channel comes with Novu's email provider, which is active by default and includes 300 credits.
+
+## Create A Notification Template
```

Comment:
We have now renamed notification template to Workflow. So:

Create A Notification Template => Create a Notification Workflow

Reply:
i based the guide from the PHP quickstart, so we will need to change "Template" to "Workflow" on that guide as well

Reply:
that has been changed already